airbyte-cdk/java/airbyte-cdk/core/src/main/java/io/airbyte/cdk/db/jdbc/AbstractJdbcCompatibleSourceOperations.java

```diff
@@ -222,7 +222,7 @@ protected void setString(final PreparedStatement preparedStatement, final int pa
   }
 
   protected void setBinary(final PreparedStatement preparedStatement, final int parameterIndex, final String value) throws SQLException {
-    preparedStatement.setBytes(parameterIndex, DatatypeConverter.parseHexBinary(value));
+    preparedStatement.setBytes(parameterIndex, DatatypeConverter.parseBase64Binary(value));
```

Comment:
Just FYI, this affects other connectors too, since this is shared code. We should understand which connectors this affects - this would be the classes with source operations that inheirit from this and classes that allow BINARY columns as cursors (which I don't think we have more of)

Reply:
I should clarify, no need to publish them - more of a documentation exercise

Reply:
They are called from postgres and mysql, and general JdbcSourceOperations which was inherited by Teradata, Oracle, Cockroach, Db2, Snowflake, Mssql, Redshift.

Indeed we do not allow binary column as cursor, both in individual SourceOperations(Mysql, Postgres) and general JDBCSourceOperations https://github.com/airbytehq/airbyte/blob/da32fc86bf6450b4d02d4cc9e5f761b80529255a/airbyte-cdk/java/airbyte-cdk/core/src/main/java/io/airbyte/cdk/db/jdbc/JdbcUtils.java#L67